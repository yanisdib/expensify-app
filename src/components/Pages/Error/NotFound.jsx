import React from 'react';
import { Link } from 'react-router-dom';

const NotFound = () => (
    <div>
        404 Not found
        <br />
        <Link to="/">Back to home</Link>
    </div>
);

export default NotFound;