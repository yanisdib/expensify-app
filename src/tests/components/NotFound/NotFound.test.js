import React from 'react';
import { shallow } from 'enzyme';
import NotFound from '../../../components/Pages/Error/NotFound';

test('should render NotFound correctly', () => {
    const wrapper = shallow(<NotFound />);
    expect(wrapper).toMatchSnapshot();
});