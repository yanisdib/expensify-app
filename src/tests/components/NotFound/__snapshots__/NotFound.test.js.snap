// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render NotFound correctly 1`] = `
<div>
  404 Not found
  <br />
  <[object Object]
    to="/"
  >
    Back to home
  </[object Object]>
</div>
`;
